packages/sui-ssr/README.md

```diff
@@ -87,12 +87,25 @@ export default {
 }
 ```
 
-Here we implement a direct log into the console. Each hook type could be or a middleware function or an array of middlewares functions.
+Here we implement a direct log into the console. Each hook type could be a middleware function or an array of middlewares functions.
 
-You can checks with hooks there are available in the hooks-types.js file.
+You can check which hooks are available in the hooks-types.js file.
 
 There are two default hooks for 404 and 500 errors. both will look for a 404.html or 500.html file in the src folder and show this file. If you dont define this files, you will get a generic error page.
 
+## Environment variables
+
+You can define environment variables by creating a yml file called `public-env.yml` in your SPA root directory:
```

Comment:
naming could `env.yml`. It's already in the public repo. Or `spa-env.yml` if it's only for SPAs